MACHINE_LEARNING/image_recognizer/image_recognizer_app/utils/unzip.py

```diff
@@ -9,11 +9,15 @@
 # accordance with the terms of the license agreement you entered into
 # with Jalasoft.
 #
+from zipfile import ZipFile
 
-import os
 
-from django.core.asgi import get_asgi_application
+class Unzip:
+    """Unzip file on same zip-file path"""
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'machine_learning.settings')
-
-application = get_asgi_application()
+    @staticmethod
+    def extract(path, filename):
```

Comment:
Please add the description of the method

Reply:
ok